from .rustplus_pb2 import AppRequest as ar
from .rustplus_pb2 import AppMessage as am
from .rustplus_pb2 import AppSendMessage as asm
from .rustplus_pb2 import AppSetEntityValue as asev
from .rustplus_pb2 import AppPromoteToLeader as aptl
from .rustplus_pb2 import AppResponse as arr
from .rustplus_pb2 import AppBroadcast as ab
from .rustplus_pb2 import AppEmpty as ae
from .rustplus_pb2 import AppSuccess as ass
from .rustplus_pb2 import AppError as aee
from .rustplus_pb2 import AppInfo as ai
from .rustplus_pb2 import AppTime as at
from .rustplus_pb2 import AppMap as ammm
from .rustplus_pb2 import AppTeamInfo as ati
from .rustplus_pb2 import AppTeamChat as atc
from .rustplus_pb2 import AppChatMessage as acm
from .rustplus_pb2 import AppEntityInfo as aei
from .rustplus_pb2 import AppEntityType as aet
from .rustplus_pb2 import AppEntityPayload as aep
from .rustplus_pb2 import AppFlag as af
from .rustplus_pb2 import AppMapMarkers as amm
from .rustplus_pb2 import Vector4 as v4
from .rustplus_pb2 import AppMarker as ammmm
from .rustplus_pb2 import AppMarkerType as amt
from .rustplus_pb2 import AppTeamChanged as atcc
from .rustplus_pb2 import AppTeamMessage as atm
from .rustplus_pb2 import AppEntityChanged as aec
from .rustplus_pb2 import AppCameraSubscribe as acs
from .rustplus_pb2 import AppCameraInput as aci


AppRequest = ar
AppMessage = am
AppSendMessage = asm
AppSetEntityValue = asev
AppPromoteToLeader = aptl
AppResponse = arr
AppBroadcast = ab
AppEmpty = ae
AppSuccess = ass
AppError = aee
AppInfo = ai
AppTime = at
AppMap = ammm
AppTeamInfo = ati
AppTeamChat = atc
AppChatMessage = acm
AppEntityInfo = aei
AppEntityType = aet
AppEntityPayload = aep
AppFlag = af
AppMapMarkers = amm
Vector4 = v4
AppMarker = ammmm
AppMarkerType = amt
AppTeamChanged = atcc
AppTeamMessage = atm
AppEntityChanged = aec
AppCameraSubscribe = acs
AppCameraInput = aci
